drop-transfer/src/ws/server/v6.rs

```diff
@@ -370,6 +370,18 @@ impl HandlerLoop<'_> {
             }
         }
     }
+
+    fn take_pause_futurues(&mut self) -> impl Future<Output = ()> {
```

Comment:
```suggestion
    fn take_pause_futures(&mut self) -> impl Future<Output = ()> {
```

But also the name is pretty cryptic, it's more like `fn finish_jobs()`

Reply:
Well, I meant to take out the future with the closing task. Hence this function is not `async` but returns owned future

Reply:
Typo fixed